addons/web/static/src/core/domain_selector/domain_selector.js

```diff
@@ -154,13 +149,18 @@ export class DomainSelector extends Component {
         this.notifyChanges();
     }
 
+    resetDomain() {
+        this.notifyChanges();
+    }
+
     updateBranchOperator(node, operator) {
         node.operator = operator;
         this.notifyChanges();
     }
 
-    async updateField(node, field) {
-        const fieldDef = await this.loadFieldDef(this.props.resModel, field);
+    async updateField(node, field, { fieldDef }) {
+        fieldDef ||= { type: "integer" };
+        field = fieldDef ? field : "";
```

Comment:
will always be truthy because of the previous line
Am I wrong?

Reply:
no, you're right :-)

Reply:
order changed